src/textures/DepthTexture.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @author Marius Kintel / https://github.com/kintel
+ */
+
+THREE.DepthTexture = function ( width, height, hasStencil ) {
+
+	var format = hasStencil ? THREE.DepthStencilFormat : THREE.DepthFormat;
+	var type = hasStencil ? THREE.UnsignedInt24_8Type : THREE.UnsignedIntType;
+
+	THREE.Texture.call( this, null, THREE.Texture.DEFAULT_MAPPING, THREE.ClampToEdgeWrapping, THREE.ClampToEdgeWrapping, THREE.NearestFilter, THREE.NearestFilter, format, type );
+
+	this.width = width;
+	this.height = height;
+	this.hasStencil = hasStencil;
```

Comment:
It's used to determine whether the depth texture has a stencil attachment. This is later used to correctly set up the GL parameters. I think we should be able to deduce these parameters from this.format and this.type and not store the hasStencil in the DepthTexture object, if that's preferable. It might be cleaner in any case.